SampleApp/build.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this available for client use, too?

Reply:
No, but it serves as an example of how to have it set up, and the changes in the readme should cover the changes they'd need to make.